Makefile

```diff
@@ -106,3 +108,26 @@ run: install ## Run the controller locally. This is for testing purposes only.
 .PHONY: clean
 clean:
 	@rm -rf $(ARTIFACTS)
+
+# Make `make test` behave just like `go test` regarding relative paths.
+test:
+	@$(MAKE) local-integration-test DEST=./internal/integration PLATFORM=linux/amd64
+	cd internal/integration && KUBECONFIG=../../kubeconfig ./integration.test -test.v
+
+talosctl:
+	curl -Lo talosctl https://github.com/talos-systems/talos/releases/download/$(TALOS_VERSION)/talosctl-$(shell uname -s | tr "[:upper:]" "[:lower:]")-amd64
+	chmod +x ./talosctl
+
+env-up: talosctl
+	./talosctl cluster create \
+		--name=cabpt-env \
+		--kubernetes-version=$(K8S_VERSION) \
+		--mtu=1450 \
+		--memory=2048 \
+		--cpus=2.0 \
```

Comment:
Both `memory` and `cpus` use default values, do we have to specify them explicitly?

Reply:
That was me fighting with the build. We can keep it or omit it, I don't thinkt it matters a lot